app/scripts/services/storage.srv.js

```diff
@@ -56,6 +56,11 @@ angular.module('dashboard')
                 else {
                     $log.error("StorageSrv: deleteKey invalid parameter");
                 }
+            },
+            reset: function () {
+                for (var key in data) {
+                    delete data[key];
```

Comment:
No biggie, just that deleteKey is for deleting one specific property. In principle replacing using an empty object sounds more simple than iterating it through.